### PR TITLE
Update for mixer models, feedbacks, presets and toggle

### PR DESCRIPTION
**Pull request by jrviz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,16 @@
 # companion-module-roland-m5000
 See HELP.md and LICENSE
+
+The M-5000 mixer uses an ethernet connection for control but this just uses the serial protocol over ip
+The older mixers have a seial connection (M-2xx, M-3xx, M-4xx) and the protocol is the same
```
Comment (josephdadams) [inline README.md]: typo `seial`

**Comment by josephdadams** [comment]
Great job!

**Comment by jrviz** [comment]
Final testing should be completed soon

**Comment by josephdadams** [comment]
It looks fine to me, I'm ready to merge it when you are.

**Comment by krocheck** [comment]
I think the product in `package.json` should be updated to an array with all the supported model.  You could add to each of the non-5000 models something like `M-200i (via serial bridge)`

**Comment by jrviz** [comment]
OK, I'll make that update. Should it be M-4xx, M-3xx or M-400, M-480, M-300, M-380 separately do you think?

**Comment by krocheck** [comment]
I would have you list the discrete models you're supporting.

**Comment by McHauge** [comment]
@krocheck if I support 20 or so supported cams in the Panasonic ptz, is that above the line for adding each, or should I add all 20 to an array?

**Comment by krocheck** [comment]
@McHauge PTZ's we handle as generic by brand, typically.  Maybe the better question here is if there are any mixer models that aren't supported?  Maybe we just change this over the module to "Roland Audio Mixer"?

**Comment by josephdadams** [comment]
@krocheck Give Roland enough time, they'll make a new one we don't support

**Comment by krocheck** [comment]
true story

**Comment by jrviz** [comment]
This supports the audio V-mixers series both current (3) and discontinued (3). Although the M-5000 was a new generation, all Roland did for remote control was expose the serial protocol to an ip port which is why it is possible to support the other mixers. So, roll on the M-6000! So Roland Audio Mixers makes sense.

**Comment by josephdadams** [comment]
I vote we rename the repo and module

**Comment by jrviz** [comment]
A slight downside of putting an array of products in the product field in the package.json is that the whole array is used for the button label in the presets tab.

**Comment by jrviz** [comment]
Tested now with mixer. Good to go.

**Pull request merged**